File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/buc/Organisation.java
package no.nav.melosys.eessi.models.buc;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class Organisation {
    private String name;
    private String countryCode;
    private String id;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/buc/Organisation.kt
package no.nav.melosys.eessi.models.buc

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class Organisation(
    var name: String? = null,
    var countryCode: String? = null, // TODO: gjør denne none-nullable
    var id: String? = null // TODO: gjør denne none-nullable
)
